src/goose.rs

```diff
@@ -1562,6 +1561,26 @@ impl Hash for GooseTask {
 mod tests {
```

Comment:
It would probably make sense to put the `mod tests` to a tests.rs?

Reply:
can be follow-up

Reply:
Currently each file has tests appropriate to that file. If we create a `tests.rs` do we then move ALL tests from all files, or just those from `goose.rs`? My concern is, I assume we will move them all, but then I wonder if this is less-organized, not more?